lib/jekyll/converters/scss.rb

```diff
@@ -179,18 +186,53 @@ def sass_configs
         )
       end
 
+      def sass_embedded_config(data)
+        {
+          :data                => data,
+          :file                => file_path,
+          :indented_syntax     => syntax == :sass,
+          :include_paths       => sass_load_paths,
+          :output_style        => sass_style,
+          :source_map          => sourcemap_required?,
+          :out_file            => output_path,
+          :omit_source_map_url => !sourcemap_required?,
+          :source_map_contents => true,
+        }
+      end
+
       def convert(content)
+        case sass_implementation
+        when "sass-embedded"
+          require "sass"
```

Comment:
TODO: Handle cases where gem is not installed or listed in Gemfile.

Reply:
Replaced `require` with `Jekyll::External.require_with_graceful_fail`